Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Build the manager binary
-FROM golang:1.13 as builder
+FROM golang:1.15 as builder
```

Comment:
Why not 1.16?

Reply:
Because it was released 2 days ago and I didn't know :)
Do you think we should update now or we can postpone it?

Reply:
go 1.16 is not in `brew` yet, so I will defer the update